health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/editVillagePopulationPopUP.js

```
@@ -47,88 +55,101 @@ const EditVillagePopulationPopUp = ({ onClose, census }) => {
-            <div className="edit-value">
-              <TextInput
-                type="text"
-                value={confirmedTotalPopulation}
-                onChange={(e) => setConfirmedTotalPopulation(e.target.value)}
-                placeholder={t(`HCM_MICROPLAN_ENTER_VILLAGE_POPULATION`)}
-              />
+    <>
+      <PopUp
+        onClose={onClose}
+        heading={t(`HCM_MICROPLAN_EDIT_POPULATION_LABEL`)}
+        children={[
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)